packages/target-ethers-v5-test/types/Events.d.ts

```diff
@@ -201,10 +227,22 @@ export class Events extends BaseContract {
   };
 
   filters: {
+    "AnonEvent1(uint256)"(
+      value1?: BigNumberish | null
+    ): TypedEventFilter<[BigNumber], { value1: BigNumber }>;
```

Comment:
Here it would be perfect to reuse the already generated type ie: 
`TypedEventFilter<AnonEvent1Event>;` but I think it's not that simple so we can ship initial implementation without this and later improve.

Reply:
I agree we should, that may involve some refactor to the TypedEventFilter generics. Maybe we can clean up the code in another PR.